examples/magento/getProductStock.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Denner\Client\MagentoClient;
+
+$config = require realpath(__DIR__ . '/../bootstrap.php');
+$params = array(
+);
+
+$client = MagentoClient::factory($config);
+
+$token = @$_GET['token'] ?: 'ajs8vvnm0vgk976iiqv6optdsjucp05r';
+$id = @$_GET['id'] ?: '050005';
+
+if (!$token) {
+    throw new RuntimeException('Missing or invalid parameter "token"');
+}
+
+$response = $client->getProductStock(
+    array(
+        'Authorization' => sprintf('Bearer %s', $token),
```

Comment:
Wouldn't it be better to handle authorization client-internally?

The client could automatically retrieve a token before the first call.

Reply:
It was also my first thought.
But then, recalling all the SOP problems that we have with the previous Magento, I decided against it, to let the operator handle the login/authorization independently.

But to integrate this in the client is still feasible 
(The main purpose for this PR was to test that the Magento2 client can be realized and test the current access to resources we'll need)